crates/validation/src/lib.rs

```diff
@@ -252,15 +393,17 @@ fn walk_transition<'a, D, L, B>(
         &'a url::Url,             // Scope.
         D::ModelDef,              // Model to validate.
         models::Id,               // Live control-plane ID.
-        models::Id,               // Assigned data-plane.
+        &'a tables::DataPlane,    // Assigned data-plane.
```

Comment:
This return type is getting pretty unwieldy, and I'm thinking that at some point it'll probably be better to declare a `TransitionOk<'a, D, L, B>` struct. Doesn't need to be now, though.